frame/examples/default-config/src/lib.rs

```diff
@@ -129,6 +126,8 @@ pub mod tests {
 		// these items are defined by frame-system as `no_default`, so we must specify them here.
 		// Note that these are types that actually rely on the outer runtime, and can't sensibly
 		// have an _independent_ default.
+		type Block = Block;
```

Comment:
The `Block` shoudl have a default, this was one of my main hopes for this line of work. Any blockers for it?

Reply:
Yeah, it requires an `UncheckedExtrinsic` that requires the `RuntimeCall`.